src/components/Chips.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render } from "src/utils/rtl";
+import { Chips } from "./Chips";
+
+describe("Chips", () => {
```

Comment:
**praise:** Oh wow! I am surprised we didn't have a test file for this. I guess we previously thought that the Storybook file was sufficient. Thanks for adding!